src/workspaces/components/Tools.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import {useSelector} from "react-redux";
+import {getTools} from "../selectors";
+import Button from "@material-ui/core/Button";
+
+type ToolProps = { workspaceUri: string }
+
+const Tools: React.FC<ToolProps> = (props: ToolProps) => {
+    const tools = useSelector(getTools);
+    const handleClick = (e : React.MouseEvent) => { e.stopPropagation() };
+    return <>
+        {tools.map( t => <Button
+            onClick={handleClick}
+            color="secondary"
+            href={t.url + "?workspace=" + props.workspaceUri}
+        >{t.label}</Button> )}
+    </>
+}
+
+export default Tools
```

Comment:
There is a code style convention used in the whole repo (basically using Prettier), but I see now that it's not set up as a dependency / executable script... So I will deal with that next week I guess.